### PR TITLE
Docs: Example of provisioning InfluxDB v2 datasource

### PR DESCRIPTION
**Pull request by wsromek**
**What this PR does / why we need it**:

Having failed to find an example of how to provision the new InfluxDB v2 source I prepared this small increment to docs.

**Special notes for your reviewer**:

### CODE REVIEW

File: docs/sources/features/datasources/influxdb.md [hunk 1]
```diff
@@ -278,11 +278,12 @@ You can now configure data sources using config files with Grafana's provisionin
 
 Here are some provisioning examples for this data source.
 
+### InfluxDB 1.x
```
Comment (oddlittlebird) [inline docs/sources/features/datasources/influxdb.md:281]:
```suggestion
### InfluxDB 1.x example
```

File: docs/sources/features/datasources/influxdb.md [hunk 2]
```diff
@@ -292,3 +293,21 @@ datasources:
     jsonData:
       httpMode: GET
 ```
+
+### InfluxDB 2.x
```
Comment (oddlittlebird) [inline docs/sources/features/datasources/influxdb.md:297]:
```suggestion
### InfluxDB 2.x example
```

**Review by oddlittlebird — CHANGES_REQUESTED** [review]
Thank you! I am not very familiar with most of the data sources, so examples like this are great additions! I have a couple heading suggestions to make them less ambiguous, but other than that it looks great.

**Comment by wsromek** [comment]
@oddlittlebird I committed the suggested changes :) 👍

**Review by oddlittlebird — APPROVED** [review]

**Comment by oddlittlebird** [comment]
Great! Once the testing completes, I will merge it.

Fun tip to save yourself some typing, if you click the little round arrows next to a reviewer's name, then Git automatically re-requests their review. 

Thanks for contributing to the documentation!

**Comment by wsromek** [comment]
:+1: I do not mind typing these couple sentences at all :) Thanks for a swift review ;)

**Pull request merged**